frontend/site/.vitepress/theme/composables/dataProxy.js

```diff
@@ -96,10 +96,6 @@ export default function useDataProxy(mapping, statuses) {
         });
     }
 
-    onMounted(() => {
-        getData();
```

Comment:
I may be missing something, but seems like `getData` is never called anymore? I had to add the onMounted back to make it work locally

Reply:
I moved it to InventoryBase but my condition is wrong without a pasted link : https://github.com/etalab/ouverture.data.gouv.fr/pull/31/files#diff-730b9322507594a4427a84fb7710a8b229714ef9e57534deb52cc4db8066b0f4R198